python/PyQt6/core/auto_generated/dxf/qgsdxfexport.sip.in

```diff
@@ -144,7 +144,7 @@ Add layers to export
 .. seealso:: :py:func:`setLayerTitleAsName`
 %End
 
-    ExportResult writeToFile( QIODevice *d, const QString &codec );  //maybe add progress dialog? other parameters (e.g. scale, dpi)?
+    ExportResult writeToFile( QIODevice *d, const QString &codec, QString *feedbackMessage = 0 );  //maybe add progress dialog? other parameters (e.g. scale, dpi)?
```

Comment:
Instead of this (which won't work in python) how about an explicit feedbackMessage() getter instead?

Reply:
@nyalldawson , done.